documentation/modules/configuring/con-config-mirrormaker2-tasks-max.adoc

```diff
@@ -35,11 +35,11 @@ spec:
   # ...
 ----
 
-For sink and source connectors, the maximum number of tasks that can run in parallel depends on a number of factors.
+For connectors, the maximum number of tasks that can run in parallel depends on a number of factors.
```

Comment:
@mimaison - I've updated the tasksMax section

Reply:
For MirrorMaker Source the maximum number of tasks is the number of partition being replicated. The number of tasks that will be started is the minimum value between that and tasks.max. There are no other factors.

For Checkpoint, it's similar but using the number of consumer groups being replicated.

For Heartbeat it's always using a single task.

I think we can remove all the rest as it's for other connectors.